consensus/core/src/authority_node.rs

```diff
@@ -110,6 +120,14 @@ impl ConsensusAuthority {
             Self::WithTonic(authority) => &authority.context,
         }
     }
+
+    #[allow(unused)]
+    fn sync_last_known_own_block_enabled(&self) -> bool {
```

Comment:
Why unused?

Reply:
It's currently being used only for testing and linter complains.

Reply:
Yeah if I do `#[cfg(test)]` then it will complain for unused `sync_last_known_own_block`  - that's why I went with allow unused in first place